src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1455,403 +1455,424 @@ public int Compare(Symbol fst, Symbol snd)
             NamespaceOrTypeSymbol qualifierOpt = null,
             LookupOptions options = default(LookupOptions))
         {
-            Debug.Assert(where != null);
-            Debug.Assert(diagnostics != null);
-
-            var symbols = result.Symbols;
-            wasError = false;
+            Symbol symbol = resultSymbol(result, simpleName, arity, where, diagnostics, suppressUseSiteDiagnostics, out wasError, qualifierOpt, options);
 
-            if (result.IsMultiViable)
+            if (symbol.Kind == SymbolKind.NamedType)
             {
-                if (symbols.Count > 1)
-                {
-                    // gracefully handle symbols.Count > 2
-                    symbols.Sort(ConsistentSymbolOrder.Instance);
-
-                    var originalSymbols = symbols.ToImmutable();
+                CheckRuntimeSupportForSymbolAccess(where, receiverOpt: null, symbol, diagnostics);
+            }
 
-                    for (int i = 0; i < symbols.Count; i++)
-                    {
-                        symbols[i] = UnwrapAlias(symbols[i], diagnostics, where);
-                    }
+            return symbol;
 
-                    BestSymbolInfo secondBest;
-                    BestSymbolInfo best = GetBestSymbolInfo(symbols, out secondBest);
+            Symbol resultSymbol(
```

Comment:
>resultSymbol [](start = 19, length = 12)

I cannot tell if the code below here has changed or not, as the diff appears to be broken.  If it has just been moved into a local function fine.  If there are other changes, can you please point them out?

Reply:
> I cannot tell if the code below here has changed or not, as the diff appears to be broken.  If it has just been moved into a local function, fine.  If there are other changes, can you please point them out?

No changes, just an extraction of a method.

---
In reply to: [264410428](https://github.com/dotnet/roslyn/pull/34012#discussion_r264410428) [](ancestors = 264410428)